notebooks/7_reshape_table_layout.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>The final sentence gets lost in this paragraph, even though I think it's something you were trying to emphasize.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>